app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
```

Comment:
The test case `(14, 14, [0, 0])` seems to be a duplicate of the edge case for age 0. Ensure that each test case is unique and tests a different scenario as per the 'Don't Repeat Yourself' principle.